### DISCUSSION TITLE
Unable to completely remove taint from compound (2in1) genetic material

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#14811] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

You're unable to fully remove taint from 2 in 1 compound genes, ie Mantis&Moloch. You can successfully remove taint in name but it will still apply tainted effects when you equip said gene.

### Reproduction steps

1. Get a 100% compound gene, in this case I used Mantis&Moloch
2. Load into a gene splicer then into a human
3. Get them killed to taint the gene
4. Combine tainted 100% compound gene with 10% Mantis gene with xenobiologist tree fully unlocked + giving yourself 200 medical skill
5. Take the supposedly untainted compound gene and slot it into gene splicer then into yourself

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested, fix coming in the next Unstable build. Closing discussion.
